recipes/tensorboard/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = "0.1.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: tensorboard
+  version: {{ version }}
+
+build:
+  # There are two wheels (py2, py3) in pypi, that seem equivalent to me.
+  # So for the time being, we try building a noarch package.
+  # Also see:
+  #  https://github.com/conda-forge/conda-smithy/pull/531
+  noarch: python
+  skip: True  # [not linux]
```

Comment:
The `skip` is incompatible with `noarch: python`. Could you elaborate on what we are trying to accomplish here?

Reply:
So should I remove completely "skip"? I'm a newbie building for noarch packages. I thought this will stop the package from building in all systems, therefore saving some time and avoiding multiple uploads when the package is accepted. On the other hand, I guess we do want to test everywhere.

I will revert that commit. @danielfrg what did you really mean when asking for "skip the build on linux"? I obviously misunderstood.